org.adempiere.base/src/org/compiere/print/ReportEngine.java

```diff
@@ -695,9 +697,10 @@ public boolean createHTML (Writer writer, boolean onlyTable, Language language,
 			table.setNeedClosingTag(false);
 			PrintWriter w = new PrintWriter(writer);
 			XhtmlDocument doc = null;
-					
+			boolean minify = MSysConfig.getBooleanValue(MSysConfig.HTML_REPORT_MINIFY, false, Env.getAD_Client_ID(getCtx()));
```

Comment:
The default here must be true.  I know in some way is useless as the SysConfig record already exists, but is better when doing maintenance to know here which is the default.